hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdatingEntryProcessor.java

```diff
@@ -96,12 +105,19 @@ public void setHazelcastInstance(HazelcastInstance hazelcastInstance) {
         this.hzInstance = hazelcastInstance;
     }
 
+    @SuppressWarnings("DuplicatedCode")
```

Comment:
instead of suppression define method creating EEC from `arguments`, `subject` and node engine, so you do not repeat security releated code

Reply:
It is duplicated for UpdatingEntryProcessor and `QueryUtil.JoinProjection`

Reply:
yes, define method for that

Reply:
Done